packages/now-static-build/test/builds.test.js

```diff
@@ -0,0 +1,182 @@
+            "options": Object {},
+            "resolve": "gatsby-plugin-vercel",
+          },
+        ],
+      }
+    `);
+  },
+  FOUR_MINUTES
+);
+
+it(
+  'Should build Gatsby with configuration but no plugins',
+  async () => {
+    const { workPath } = await runBuildLambda(
+      path.join(__dirname, 'build-fixtures/02-gatsby-user-config')
+    );
+
+    const contents = await fs.readdir(workPath);
+
+    expect(contents.some(name => name === 'gatsby-config.js')).toBeTruthy();
+    expect(
+      contents.some(
+        name => name === 'gatsby-config.js.__vercel_builder_backup__.js'
+      )
+    ).toBeTruthy();
+
+    expect(require(path.join(workPath, 'gatsby-config.js')))
+      .toMatchInlineSnapshot(`
+      Object {
+        "plugins": Array [
+          Object {
+            "options": Object {},
+            "resolve": "gatsby-plugin-vercel",
+          },
+        ],
+        "siteMetadata": Object {
+          "author": "@gatsbyjs",
+          "description": "Kick off your next, great Gatsby project with this default starter. This barebones starter ships with the main Gatsby configuration files you might need.",
+          "title": "Gatsby Default Starter",
+        },
+      }
+    `);
+  },
+  FOUR_MINUTES
+);
+
+it(
+  'Should build Gatsby with configuration that has plugins',
+  async () => {
+    const { workPath } = await runBuildLambda(
+      path.join(__dirname, 'build-fixtures/03-gatsby-with-plugins')
+    );
+
+    const contents = await fs.readdir(workPath);
+
+    expect(contents.some(name => name === 'gatsby-config.js')).toBeTruthy();
+    expect(
+      contents.some(
+        name => name === 'gatsby-config.js.__vercel_builder_backup__.js'
+      )
+    ).toBeTruthy();
+
+    expect(require(path.join(workPath, 'gatsby-config.js')))
+      .toMatchInlineSnapshot(`
+      Object {
+        "plugins": Array [
+          "gatsby-plugin-react-helmet",
+          Object {
+            "options": Object {},
+            "resolve": "gatsby-plugin-vercel",
+          },
+        ],
+        "siteMetadata": Object {
+          "author": "@gatsbyjs",
+          "description": "Kick off your next, great Gatsby project with this default starter. This barebones starter ships with the main Gatsby configuration files you might need.",
+          "title": "Gatsby Default Starter",
+        },
+      }
+    `);
+  },
+  FOUR_MINUTES
+);
+
+it(
+  'Should build Gatsby with configuration that already has plugin (simple)',
+  async () => {
+    const { workPath } = await runBuildLambda(
+      path.join(__dirname, 'build-fixtures/04-gatsby-no-dupe-simple')
+    );
+
+    const contents = await fs.readdir(workPath);
+
+    expect(contents.some(name => name === 'gatsby-config.js')).toBeTruthy();
+    expect(
+      contents.some(
+        name => name === 'gatsby-config.js.__vercel_builder_backup__.js'
+      )
+    ).toBeTruthy();
+
+    expect(require(path.join(workPath, 'gatsby-config.js')))
+      .toMatchInlineSnapshot(`
```

Comment:
Why toMatchInlineSnapshot instead of deepEqual?

Reply:
Because snapshots were easier. 😄